client/src/main/kotlin/no/nav/su/se/bakover/client/dokarkiv/JournalpostRequest.kt

```diff
@@ -97,6 +98,47 @@ sealed class Journalpost {
             )
         )
     }
+
+    data class Trukket(
+        val person: Person,
+        val pdf: ByteArray,
+        val sakId: UUID,
+        val søknadId: UUID,
+    ) : Journalpost() {
+        override val tittel: String = "Vedtak om trekking av søknad for supplerende stønad"
+        override val avsenderMottaker: AvsenderMottaker = AvsenderMottaker(
+            id = person.ident.fnr.toString(),
+            navn = søkersNavn(person)
+        )
+        override val bruker: Bruker = Bruker(id = person.ident.fnr.toString())
+
+        override val sak: Fagsak = Fagsak(sakId.toString())
+
+        override val journalpostType: JournalPostType = JournalPostType.UTGAAENDE
+        override val kanal: String? = null
+        override val journalfoerendeEnhet: String = "4815"
+        override val dokumenter: List<JournalpostDokument> = toJournalpostDokument()
+
+        private fun toJournalpostDokument() = listOf(
+            JournalpostDokument(
+                tittel = tittel,
+                dokumentKategori = DokumentKategori.VB,
```

Comment:
Tipper denne isåfall burde være noe annet enn `VB` også (som står for vedtaksbrev). Sjekk med fag!

Reply:
👍